arviz/plots/kdeplot.py

```diff
@@ -262,6 +262,7 @@ def plot_kde(
     return ax
 
 
+@njit(cache=True)
```

Comment:
Is this really faster? Last time I tried this (and if my memory is not failing me) this was slower/or did not give any speed-up. That's why numba was restricted to  `_histogram`. It will be nice if every PR related to numba comes with  the speed-up. Maybe we should also have test that ensure the _numbified_ function is faster than the non-modified one.

Reply:
That sounds interesting.

Reply:
I agree, the improvement isn't much and sometimes the function is a bit slower. This is what I wanted to get checked out at first so as to ensure that the lack of speed-up was not because of some error in the code. Regarding tests, I agree that it is an interesting idea.